tensorflow/python/keras/callbacks.py

```diff
@@ -1723,6 +1723,8 @@ def _log_weight_as_image(self, weight, weight_name, epoch):
         w_img = array_ops.transpose(w_img, perm=[2, 0, 1])
         shape = K.int_shape(w_img)
       w_img = array_ops.reshape(w_img, [shape[0], shape[1], shape[2], 1])
+    elif len(shape) != 4:
+      raise ValueError('Invalid shape: {}'.format(repr(shape)))
```

Comment:
It's actually common for people to want to visualize whatever weights of their Model can be visualized, while still having weights that can't be visualized. I think silently skipping weights that can't be visualized is the right approach here

Reply:
So, can we update the document then?